Core/Utilities/SiteCEMapping.py

```diff
@@ -7,124 +7,8 @@
-
-#############################################################################
-
-
-def getSiteCEMapping(gridName=None):
-  """ Returns a dictionary of all sites and their CEs as a list, e.g.
-      {'LCG.CERN.ch':['ce101.cern.ch',...]}
-      If gridName is specified, result is restricted to that Grid type.
-  """
-  sites = getSites(gridName=gridName)
-  if not sites['OK']:
-    return sites
-
-  siteCEMapping = {}
-  for candidate in sites['Value']:
-    grid = candidate.split('.')[0]
-    candidateCEs = gConfig.getValue('/Resources/Sites/%s/%s/CE' % (grid, candidate), [])
-    if candidateCEs:
-      siteCEMapping[candidate] = candidateCEs
-    else:
-      gLogger.debug('No CEs defined for site %s' % candidate)
-
-  return S_OK(siteCEMapping)
-
-#############################################################################
-
-
-def getCESiteMapping(gridName=None):
-  """ Returns a dictionary of all CEs and their associated site, e.g.
-      {'ce101.cern.ch':'LCG.CERN.ch', ...]}
-      Assumes CS structure of: /Resources/Sites/<GRIDNAME>/<SITENAME>
-  """
-  sites = getSites(gridName=gridName)
-  if not sites['OK']:
-    return sites
-
-  ceSiteMapping = {}
-  for candidate in sites['Value']:
-    grid = candidate.split('.')[0]
-    siteCEs = gConfig.getValue('/Resources/Sites/%s/%s/CE' % (grid, candidate), [])
-    for ce in siteCEs:
-      if ce in ceSiteMapping:
-        current = ceSiteMapping[ce]
-        gLogger.error('CE %s already has a defined site %s but it is also defined for %s' % (ce, current, candidate))
-      else:
-        ceSiteMapping[ce] = candidate
-
-  return S_OK(ceSiteMapping)
-
-#############################################################################
-
-
-def getSiteForCE(computingElement):
-  """ Given a Grid CE name this method returns the DIRAC site name.
-
-      WARNING: if two or more sites happen to have the same ceName/queueName, then only the first found is returned
-  """
-  sites = getSites()
-  if not sites['OK']:
-    return sites
-
-  for candidate in sites['Value']:
-    grid = candidate.split('.')[0]
-    siteCEs = gConfig.getValue('/Resources/Sites/%s/%s/CE' % (grid, candidate), [])
-    if computingElement in siteCEs:
-      finalSite = candidate
-      return S_OK(finalSite)
-  # FIXME: this is strange but this was how it was coded
-  return S_OK('')
-
-#############################################################################
-
-
-def getCEsForSite(siteName):
-  """ Given a DIRAC site name this method returns a list of corresponding CEs.
-  """
-  if not re.search('.', siteName):
-    return S_ERROR('%s is not a valid site name' % siteName)
-  gridName = siteName.split('.')[0]
-  return S_OK(gConfig.getValue('/Resources/Sites/%s/%s/CE' % (gridName, siteName), []))
-
-#############################################################################
+from DIRAC.ConfigurationSystem.Client.Helpers.Resources import getCESiteMapping
 
 
 def getQueueInfo(ceUniqueID, diracSiteName=''):
```

Comment:
This function is used only in one place - dirac-wms-cpu-normalization command. I would move this function to the command itself and drop this entire file.

Reply:
Done.